data/mustafa-edu.json

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
There is a blank line here. It doesn't impact the JSON, but we might want to remove it.